cmake_format/configuration.py

```diff
@@ -172,6 +177,14 @@ def __init__(self, line_width=80, tab_size=2,
     # stack is this config object... so I'm abusing it by adding this field here
     self.first_token = None
 
+  def __setattr__(self, key, value):
```

Comment:
This seems unnecessary. We can just compile the regex at the format phase and avoid overriding this builtin.

Reply:
It's been a while, but I think I did that after-the-fact because I kept failing tests when I would update the min length but forgot to update the compiled regex, but didn't want to do it for every line (didn't know it could be setup where you are now :))